app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -85,6 +88,17 @@ fun BottomAppBarAll(
     val totalUnreads = unreadCounts?.let { unreadCountTotal(it) }
 
     if (showBottomNav == true) {
+        val window = (LocalContext.current as Activity).window
+        val colorScheme = MaterialTheme.colorScheme
+
+        DisposableEffect(Unit) {
```

Comment:
I'm guessing the check failed because of a missing space between `DisposableEffect` and the following parenthesis.